wire-protoc-tests/src/test/java/com/squareup/wire/ProtocWireTests.kt

```diff
@@ -0,0 +1,54 @@
+/*
```

Comment:
I think the class pass under `test` should be `kotlin` instead of `java`. WDYT?

Reply:
I don't think it matters. We have kotlin code under `java` all over the place for now.